api/functions.py

```diff
@@ -682,12 +765,27 @@ def update(id: int, name: str, deliver_service: str) -> DistributorDantic:
 # noinspection PyTypeChecker
 class SuppliesCrud:
     @staticmethod
-    def add(component_id: int, count: float, distributor: int):
+    def add(component_id: int, count: float, distributor_id: int) -> OutputSupplyDantic:
         sess = Session()
-        supply = Supplies(component_id=component_id, count=count, distributor=distributor)
-        sess.add(supply)
-        sess.commit()
-        sess.close()
+        try:
+            supply = Supplies(component_id=component_id, count=count, distributor=distributor_id)
+            print(1)
```

Comment:
Check out https://github.com/gruns/icecream 🍧
It's very cool and you can disable debug output.